pkg/3scale/amp/component/redis.go

```diff
@@ -297,6 +297,13 @@ func (redis *Redis) BackendRedisSecret() *v1.Secret {
 			BackendSecretBackendRedisStorageSentinelRoleFieldName:  redis.Options.BackendRedisStorageSentinelRole,
 			BackendSecretBackendRedisQueuesSentinelHostsFieldName:  redis.Options.BackendRedisQueuesSentinelHosts,
 			BackendSecretBackendRedisQueuesSentinelRoleFieldName:   redis.Options.BackendRedisQueuesSentinelRole,
+			// TLS
+			BackendSecretBackendRedisSslCa:   redis.Options.BackendConfigSslCa,
```

Comment:
so this is used for when internal Redis is used. IMO this shouldn't be here since we are planning on removing internal Redis therefore, creation of Redis secret will depend on customer creating it, operator will no longer be creating it. However, good to clarify it with the group.

Reply:
How it's differ from existed NON-TLS fields?...
Meanwhile, while we still creating secrets - it just can help have valid secrets with all required fields

Reply:
thanks for comment. But I'm not sure, as if to go this way - the previouse code (lines 299 and before) - also not needed. Also - TLS is not enabled if secret fields related to TLS are empty.